lib/linux_admin/process/memory.rb

```diff
@@ -0,0 +1,49 @@
+module LinuxAdmin
+  module Process
+    class Memory
+      attr_reader :pss, :rss, :uss, :shared, :swap, :size
+
+      def initialize(pid = Process.pid, smaps = nil)
```

Comment:
One more thing...won't Process here resolve to the wrong module because you're already nested in a module named Process.